sdks/java/io/azure/src/test/java/org/apache/beam/sdk/io/azure/blobstore/AzfsTestUtils.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.sdk.io.azure.blobstore;
+
+import com.azure.storage.blob.BlobServiceClient;
+import org.apache.beam.sdk.io.azure.options.BlobstoreOptions;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.mockito.Mockito;
+
+class AzfsTestUtils {
+  static BlobstoreOptions azfsOptions() {
+    BlobstoreOptions options = PipelineOptionsFactory.as(BlobstoreOptions.class);
+    options.setAzureConnectionString(System.getenv("AZURE_STORAGE_CONNECTION_STRING"));
```

Comment:
It assumes that the running env has `AZURE_STORAGE_CONNECTION_STRING`, which should be false for most of time.

Reply:
I had been running these tests as integration tests with my Azure account. Since Beam does not have an Azure account as of now, I will convert these tests to unit tests and I will move integration tests to a separate file.